hypha/static_src/src/sass/apply/components/_status-bar.scss

```diff
@@ -26,189 +31,136 @@
     &__subheading {
         display: inline-block;
         padding: 5px 10px;
-        margin: 10px 0 0;
         color: $color--white;
         background-color: $color--tomato;
     }
 
-    &__icon {
-        position: absolute;
-        top: -10px;
-        left: 0;
-        z-index: 30;
-        display: none;
-        width: 20px;
-        height: 20px;
-
-        .status-bar__item--is-current &,
-        .status-bar__item--is-complete & {
-            display: block;
-
-            .status-bar--small & {
-                display: block;
-                border-radius: 50%;
-                box-shadow: 0 1px 9px 0 $color--black-50;
-            }
-        }
-
-        .status-bar__item:first-of-type & {
-            left: -10px;
-        }
+    &__text {
+        font-size: map-get($font-sizes, milli);
+        font-weight: $weight--bold;
     }
 
     &__item {
-        position: relative;
         flex: 1;
-        height: 3px;
-        background: $color--mid-grey;
+        position: relative;
+        padding-block-start: var(--tooltip-margin-top);
+
+        // The bar for each item.
+        border-block-start: var(--bar) solid $color--mid-grey;
 
-        // every items dot
+        // The dot for each item.
         &::before {
+            display: flex;
+            place-items: center;
             position: absolute;
-            top: -10px;
-            left: 0;
-            width: 20px;
-            height: 20px;
+            top: calc(-1 * var(--dot) / 2);
+            left: calc(-1 * var(--dot) / 2);
+            width: var(--dot);
+            height: var(--dot);
             background: $color--dark-grey;
-            border: 5px solid $color--mid-grey;
+            border: calc(var(--dot) / 4) solid $color--mid-grey;
             border-radius: 50%;
             content: "";
 
-            .status-bar--small & {
+            #{$root}--small & {
                 background: $color--white;
             }
         }
 
-        // last items dont have a dot
-        &:last-of-type {
-            flex: 0;
-            height: 0;
+        &--is-current {
+            &::before {
+                background: $color--white;
+                border-color: $color--tomato;
+            }
 
-            &.status-bar__item--is-complete {
-                &::after {
-                    display: none;
-                }
+            // Fill the bar all the way on accepted/declined.
+            &:last-of-type {
+                border-color: $color--primary;
             }
         }
 
-        &:first-of-type {
+        &--is-complete {
+            border-color: $color--primary;
+
             &::before {
-                left: -10px;
+                font-size: map-get($font-sizes, zeta);
+                font-weight: bold;
+                background: $color--primary;
+                border-color: $color--primary;
+                content: "✓";
```

Comment:
All looks good! The only super small thing I saw on test was that the applicant dashboard's status bar still had a black color for the check. Might be a good idea to keep the additions of [49ffdbd](https://github.com/HyphaApp/hypha/pull/3672/commits/49ffdb8d722c9d0337b1f59bf2b9176abeeddae4)

```suggestion
                content: "✓";
                color: $color--white;
```

Reply:
<img width="856" alt="Screenshot 2024-01-11 at 09 33 05" src="https://github.com/HyphaApp/hypha/assets/145372368/96c16783-3ef8-4ef6-ac3c-58e7b0d72e23">

applicant dashboard in question ^

Reply:
Missed that one, thanks for testing!